syncd/VendorSai.cpp

```diff
@@ -721,7 +721,7 @@ sai_status_t VendorSai::bulkRemove(
             break;
 
         default:
-            SWSS_LOG_ERROR("not implemented %s, FIXME", sai_serialize_object_type(object_type).c_str());
+            SWSS_LOG_ERROR("not implemented, FIXME");
             return SAI_STATUS_NOT_IMPLEMENTED;
```

Comment:
please bring back ll the changes in VendorSai.cpp i fixed those in other pr especially SAI_OBJECT_TYPE_NEXT_HOP_GROUP_MEMBER